rs/nns/sns-wasm/tests/deploy_new_sns.rs

```diff
@@ -221,7 +221,10 @@ fn test_deploy_cleanup_on_wasm_install_failure() {
             error: Some(SnsWasmError {
                 message: "Error installing Governance WASM: Failed to install WASM on canister \
                 qsgjb-riaaa-aaaaa-aaaga-cai: error code 5: Error from Canister qsgjb-riaaa-aaaaa-aaaga-cai: \
-                Canister called `ic0.trap` with message: did not find blob on stack.\n\
+                Canister called `ic0.trap` with message: did not find blob on stack\nCanister Backtrace:\n\
+                unknown function at index 672\n\
+                unknown function at index 59\n\
+                unknown function at index 451\n.\n\
```

Comment:
I see a test failure related to this in CI. Perhaps it's best to avoid precise checks for indices? I suppose these will need to be changed whenever there is a change in the code fo SNS Wasm canister.

Reply:
Good point, I changed it to remove assertions about the backtrace.